### DISCUSSION TITLE
Feedback: move location marker to bottom in Android Auto view

### DISCUSSION THREAD

**Discussion by guerda** [osmandapp/OsmAnd#13462] [Ideas]
I really like the new Android Auto feature of OsmAnd.
I have a small feedback on the new map view: currently the location marker is centered in the map. While driving the car, everything behind us is not really interesting, so I would suggest to move the location marker to the bottom. This makes more map content visible which is lying in front of the car.

I hope this makes sense.

Best
Philip

**Comment by vshcherb** [comment]
That's strange. I think it's already at the bottom probably you need to configure setting display my location always  in the center or change map orientation
![telegram-cloud-photo-size-4-5947340179863876983-y](https://user-images.githubusercontent.com/1042025/149423247-690148df-0d7a-472f-96cd-97746426b448.jpg)

**Reply by guerda** [reply]
I haven't changed anything in the default configuration, to be honest. I'll try with the latest version again. 
Are there Android Auto specific settings somewhere? I was not able to find them in the app yet.

**Comment by alex-osm** [comment]
@guerda Please try to change compass mode or start navigation.

**Reply by guerda** [reply]
I did observe it during navigation, but I will try again. 

Besides this, is this the right place to give feedback about OsmAnd?

**Reply by vshcherb** [reply]
Yes, Github Discussions for Q&A / General, Issues for bugs that we understand & reproduce.

**Reply by guerda** [reply]
I double checked and it is working as you described. During navigation it's on the bottom, without navigation it's centered.
